Templates/integration_config_template.ps1

```diff
@@ -0,0 +1,24 @@
+<#
+.Synopsis
+   DSC Configuration Template for DSC Resource Integration tests.
+.DESCRIPTION
+   To Use:
+     1. Copy to \Tests\Integration\ folder and rename MSFT_x<ResourceName>.config.ps1 (e.g. MSFT_xFirewall.config.ps1)
+     2. Customize TODO sections.
+
+.NOTES
+#>
+
+
+# TODO: Modify ResourceName
```

Comment:
We should add in a dot source to the 'MSFT_<xResourceName>'.ps1 to load in the config data for the integration tests.